server/src/test/java/io/crate/integrationtests/SnapshotRestoreIntegrationTest.java

```diff
@@ -171,8 +212,8 @@ public void testCreateSnapshot() throws Exception {
         assertThat(response.rowCount(), is(1L));
 
         execute("select name, \"repository\", concrete_indices, state from sys.snapshots order by 2");
-        assertThat(TestingHelpers.printedTable(response.rows()),
-            is(String.format(
+        assertThat(printedTable(response.rows()),
+                   is(String.format(
                 "my_snapshot| my_repo| [%s.backmeup]| SUCCESS\n" +
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.

Reply:
This is testing code and the use of `\n` instead of `%n` is okay here. The results are never actually printed out or written to a file or anything.

Reply:
noticed.